articles/azure/azs-how-use-azure-api.md

```diff
@@ -36,62 +38,50 @@ To obtain an access token:
 
 1. Send a POST request to the Azure REST authentication/login endpoint:
 
-    `https://login.microsoftonline.com/<tenant_id>/oauth2/token`
+    `https://login.microsoftonline.com/<tenant_id>/oauth2/token`.
 
-    where `tenant_id` is one of:
+    where **tenant_id** is one of:
 
-    - Your tenant domain, for example, `ukcloud.onmicrosoft.com`, `ukcloud.com`, `example.mydomain.com`
+    - Your tenant domain, for example, `ukcloud.onmicrosoft.com`, `ukcloud.com`, `example.mydomain.com`.
```

Comment:
for example;

Reply:
Changed to: `Your tenant domain, For example: `ukcloud.onmicrosoft.com`, `ukcloud.com`, `example.mydomain.com`.`